docs/_templates/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+.. _quick_start:
+
+Quick Start
+===========
+
+This quick-start contains an overview of how the ``xpdAn`` software works.
+To understand more details about it, please refer to the detailed documentation in :ref:`xpdu`
+
+Please use this page as a reminder of the workflow and to copy & paste code snippets into the
+ipython terminals that are controlling your experiment and analysis.  After
+pasting, hit enter.
+
+Remember, to post questions about anything XPD, including software, and to see archived answers, at the `XPD-Users Google group
+<https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . If you are not already a member please request to join
+the community
+
+XPD Data Analysis Workflow Summary
+------------------------------------
+
+This is the summary of the steps for analysis data at XPD. They are explained below.
+Carry out the steps in this order to ensure a successful analysis.
+
+  1. If you haven't already, join `XPD-Users Google group <https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . Look here for answers if you get stuck and if the answer to your question is not already there, please ask it!
+
+  2. If not started already start a proxy and an analysis session.
+  To start the proxy run ``bluesky-0MQ-proxy {{ bs_proxy_config }}``.
```

Comment:
how do I "run" something?  Do I have to type this somewhere?  If so, where?

Keep it brief, but a few more words here could help

Reply:
Actually it seems that we won't need this, since it will be running in the background automatically.
Attn: @danielballan